built-in-references/Kubernetes/allowed-external-ips/template.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazureexternalips
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureExternalIPs
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            allowedIPs:
+              type: array
+              items:
+                type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sexternalips
+
+        violation[{"msg": msg}] {
+          input.review.kind.kind == "Service"
+          input.review.kind.group == ""
+          allowedIPs := {ip | ip := input.parameters.allowedIPs[_]}
+          externalIPs := {ip | ip := input.review.object.spec.externalIPs[_]}
+          forbiddenIPs := externalIPs - allowedIPs
+          count(forbiddenIPs) > 0
+          msg := sprintf("service has forbidden external IPs: %v", [forbiddenIPs])
```

Comment:
This message should read `"Service has forbidden external IPs: %v. Allowed IPs: %v"`

Reply:
As we discussed here https://github.com/Azure/Community-Policy/pull/152#discussion_r543836768, currently we can keep it. If a user asks for the allowed IP to be returned, we will revisit it.